src/test/resources/application-test.yml

```diff
@@ -0,0 +1,23 @@
+server:
```

Comment:
do we really need this config? most of it is duplication of application.yml. Difference is not required for tests, isn't it?

Reply:
removed